cmd/kubeops/cmd/root.go

```diff
@@ -52,8 +52,12 @@ func init() {
 	setFlags(rootCmd)
 }
 
+// TODO(agamez): remove this after the v2.4.6 release
+var assetsvcurlDeprecated string
+
 func setFlags(c *cobra.Command) {
-	c.Flags().StringVar(&serveOpts.AssetsvcURL, "assetsvc-url", "https://kubeapps-internal-assetsvc:8080", "URL to the internal assetsvc")
+	// TODO(agamez): remove 'assetsvc-url' after the v2.4.6 release
+	c.Flags().StringVar(&assetsvcurlDeprecated, "assetsvc-url", "", "(DEPRECATED)")
```

Comment:
This is an ugly workaround to get the upgrade test passing... otherwise (see my previous comment), they're gonna fail in both our CI and Bitnami's.
Once we release v2.4.6, we can safely delete it

Reply:
I'm adding a card to our project dashboard... I can't rely on my memory 👴